app/src/lib/stores/app-store.ts

```diff
@@ -484,6 +487,12 @@ export class AppStore extends TypedBaseStore<IAppState> {
 
     this.showWelcomeFlow = !hasShownWelcomeFlow()
 
+    // If the user never selected whether to use Windows OpenSSH or not, use it
+    // by default if we have to show the welcome flow (i.e. if it's a new install)
+    if (__WIN32__ && getBoolean(UseWindowsOpenSSHKey) === undefined) {
+      this._setUseWindowsOpenSSH(this.showWelcomeFlow)
+    }
```

Comment:
This is what I used to check whether it's a first install or not: whether the welcome flow has been shown or not. If there is no stored value for this setting, set it to `true` only if we still have to show the welcome flow (i.e. if it's the first time the user opens the app).